webpack/prod.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     },
 
     output: {
-        path: path.resolve(__dirname, '../resources/web/snd/app/'),
+        path: path.resolve(__dirname, '../resources/web/snd/gen/app/'),
```

Comment:
@labkey-susanh Any specific reason to leave in this /app subdir within /gen?

Reply:
Not from my perspective.  Just trying to be minimally invasive.  @labkey-martyp Any objections to dropping the "app" directory?

Reply:
Sorry missed this.  No I don't think we need the app subdir